client/my-sites/plans-comparison/plans-comparison-features.tsx

```diff
@@ -156,9 +156,9 @@ export const planComparisonFeatures: PlanComparisonFeature[] = [
 		},
 	},
 	{
-		title: translate( 'Storage space' ),
+		title: translate( 'Storage' ),
 		description: translate(
-			'The free plan allows a maximum storage of 500MB, which equals to approximately 100 high quality images, whilst with Managed WordPress you may go all the way up to 50GB, enough space for 10,000 high quality images of the same size.'
+			'The free plan allows a maximum storage of 500MB, which equals to approximately 100 high quality images, whilst with Pro you may go all the way up to 50GB, enough space for 10,000 high quality images of the same size.'
```

Comment:
Maybe change "whilst" to "while" here also (since this is already going to need to be re-translated, and it's better not to have to re-translate it again later)?

As far as I know, the base text in Calypso should always be using American English, given that there is a separate language option for British English, and in theory the "translators" could use "whilst" for that one if they wanted :smile: 

![Screenshot from 2022-03-15 10-48-45](https://user-images.githubusercontent.com/235183/158404720-298f26fe-ddf5-42ba-977d-576d850874e6.png)